index.html

```diff
@@ -12310,8 +12310,51 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<div class="property" id="aria-orientation">
 			<pdef>aria-orientation</pdef>
 			<div class="property-description">
-				<p><a>Indicates</a> whether the element's orientation is horizontal, vertical, or unknown/ambiguous.</p>
-				<p class="note">In ARIA 1.1, the default value for <pref>aria-orientation</pref> changed from <code>horizontal</code> to <code>undefined</code>. Implicit defaults are defined on some roles (e.g., <rref>slider</rref> defaults to horizontal; <rref>scrollbar</rref> defaults to vertical) but remain undefined on roles where an expected default orientation is ambiguous (e.g., <rref>radiogroup</rref>).</p>
+				<p>The <pref>aria-orientation</pref> attribute <a>indicates</a> whether the element's orientation is horizontal, vertical, or role-dependent/unknown/ambiguous.</p>
+
+			    <p>The <pref>aria-orientation</pref> attribute is an <a>enumerated attribute</a> with the following keywords and states:
+				<table>
+					<thead>
+						<tr>
+							<th scope="col">Keyword</th>
+							<th scope="col">State
+							<th scope="col">Description</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr>
+							<th class="value-name" scope="row"><code>horizontal</code></th>
+							<td>horizontal</td>
+							<td class="value-description">The element is oriented horizontally.</td>
+						</tr>
+						<tr>
+							<th class="value-name" scope="row"><code>undefined</code></th>
+							<td>undefined</td>
+							<td class="value-description">The element's orientation is unknown/ambiguous.</td>
+						</tr>
+						<tr>
+							<th class="value-name" scope="row"><code>vertical</code></th>
+							<td>vertical</td>
+							<td class="value-description">The element is oriented vertically.</td>
+						</tr>
+					</tbody>
+				</table>
+
+			    <p>The <pref>aria-orientation</pref> attribute's <a>invalid value default</a> and <a>missing value default</a> are the undefined state.
+
+				<p class="note">In ARIA 1.1, the default value for <pref>aria-orientation</pref> changed from <code>horizontal</code> to <code>undefined</code>.</p>
+
+				<p>The state of the <pref>aria-orientation</pref> property of an element <var>element</var> is computed as follows:
```

Comment:
Now that you added the algorithm the "explicit value" for aria-orientation is in two places, the algorithm and the characteristic/value table for the relevant roles (see https://w3c.github.io/aria/#listbox). Maybe it doesn't need to be in two places? Or maybe for spec understandablity, it should be?

Reply:
You mean "Implicit Value for Role"? Yeah, I think we could remove that if we go down this path.